deblur/deblurring.py

```diff
@@ -43,7 +43,10 @@ def get_sequences(input_seqs):
         If all the sequences do not have the same length either aligned or
         unaligned.
     """
-    seqs = [Sequence(id, seq) for id, seq in input_seqs]
+    try:
+        seqs = [Sequence(id, seq) for id, seq in input_seqs]
+    except:
+        seqs = []
```

Comment:
What do you think about print out a message in the log file, indicating that there were no sequences detected?

@amnona what do you think?